fs_storage/readme/USAGE.md

```diff
@@ -75,6 +75,8 @@ for the following fields:
 - options
 - directory_path
 
+When evaluating directory_path, `{db_name}` is replaced by the database name.
```

Comment:
@vincent-hatakeyama Can you explain the use case where it could be a food practice to use the `{db_name}` placeholder in your directory path?